service/host.go

```diff
@@ -95,7 +95,7 @@ func (s *host) IsRunning() bool {
 func (s *host) OnCriticalError(err error) {
 	shutdown := true
 	if s.observer == nil {
-		s.Logger().Warn(
+		s.log.Warn(
```

Comment:
If we decide on using the ulog.Logger, maybe we should be consistent and use it everywhere in our code. We don't have access to context here but so do service owners in some cases and we still expect them to inject a context.